services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala

```diff
@@ -51,7 +50,7 @@ class WorkbenchHealthMonitorServiceActor(val serviceConfig: Config, globalConfig
   private def checkGcs(): Future[SubsystemStatus] = {
     // For any expected production usage of this check, the GCS bucket should be public read */
     val gcsBucketToCheck = serviceConfig.as[String]("gcs-bucket-to-check")
-    val storage = googleAuth.credential(WorkflowOptions.empty) map { c => GcsStorage.gcsStorage(googleConfig.applicationName, c) }
+    val storage = Future(googleAuth.credential(Map.empty)) map { c => GcsStorage.gcsStorage(googleConfig.applicationName, c) }
```

Comment:
Will these wind up retrying? I don't think so but maybe it's lurking somewhere underneath. Since these are firing off fairly regularly anyways I don't think retrying here would be right.

Reply:
I thought also that retrying would _not_ be right in this particular case for the same reasons.

These no longer retry. They were previously, via an implicit actor system being sent in to the old methods. That logic has been separated into a different wrapper.